x/slashing/keeper.go

```diff
@@ -70,7 +71,12 @@ func (k Keeper) handleDoubleSign(ctx sdk.Context, addr crypto.Address, infractio
 	revisedFraction := k.capBySlashingPeriod(ctx, consAddr, fraction, distributionHeight)
 	logger.Info(fmt.Sprintf("Fraction slashed capped by slashing period from %v to %v", fraction, revisedFraction))
 
-	// Slash validator
+	// Slash validator.
+	// `power` is the int64 power of the validator as provided to/by
+	// Tendermint.  This value is validator.Tokens as sent to Tendermint via
+	// ABCI, and now received as evidence.
+	// The revisedFraction (which is the new fraction to be slashed) is passed
+	// in separately to separately slash unbonding and rebonding delegations.
```

Comment:
Improper use of capitals and periods

Reply:
Removed some periods, not sure which capitals you're referring to.